bridge-token-factory/tests/token_transfer.rs

```diff
@@ -42,33 +48,123 @@ fn setup_token_factory() -> (UserAccount, ContractAccount<BridgeTokenFactoryCont
             PROVER.to_string(), LOCKER_ADDRESS.to_string()
         )
     );
-    let alice = root.create_user(ALICE.to_string(), to_yocto("100"));
 
     (root, factory)
 }
 
+#[derive(BorshSerialize)]
+struct AugmentedProof {
+    proof: Proof,
+    skip_call: bool,
+}
+
+impl From<Proof> for AugmentedProof {
+    fn from(proof: Proof) -> Self {
+        Self {
+            proof,
+            skip_call: false,
+        }
+    }
+}
+
+#[macro_export]
+macro_rules! call_json {
+    ($signer:expr, $contract:ident, $method:ident, $arg:tt, $gas:expr, $deposit:expr) => {
+        $signer.call(
+            $contract.clone(),
+            stringify!($method),
+            json!($arg).to_string().into_bytes().as_ref(),
+            $gas,
+            $deposit,
+        )
+    };
+    ($signer:expr, $contract:ident.$method:ident($arg:tt), $gas:expr, $deposit:expr) => {
+        call_json!($signer, $contract, $method, $arg, $gas, $deposit)
+    };
+    ($signer:expr, $contract:ident.$method:ident($arg:tt)) => {
+        call_json!(
+            $signer,
+            $contract,
+            $method,
+            $arg,
+            near_sdk_sim::DEFAULT_GAS,
+            near_sdk_sim::STORAGE_AMOUNT
+        )
+    };
+    ($signer:expr, $contract:ident.$method:ident($arg:tt), deposit=$deposit:expr) => {
+        call_json!(
+            $signer,
+            $contract,
+            $method,
+            $arg,
+            near_sdk_sim::DEFAULT_GAS,
+            $deposit
+        )
+    };
+}
+
+#[macro_export]
+macro_rules! call_borsh {
+    ($signer:expr, $contract:ident, $method:ident, $arg:expr, $gas:expr, $deposit:expr) => {
+        $signer.call(
+            $contract.clone(),
+            stringify!($method),
+            &$arg.try_to_vec().unwrap(),
+            $gas,
+            $deposit,
+        )
+    };
+    ($signer:expr, $contract:ident.$method:ident($arg:expr), $gas:expr, $deposit:expr) => {
+        call_borsh!($signer, $contract, $method, $arg, $gas, $deposit)
+    };
+    ($signer:expr, $contract:ident.$method:ident($arg:expr)) => {
+        call_borsh!(
+            $signer,
+            $contract,
+            $method,
+            $arg,
+            near_sdk_sim::DEFAULT_GAS,
+            near_sdk_sim::STORAGE_AMOUNT
+        )
+    };
+    ($signer:expr, $contract:ident.$method:ident($arg:expr), deposit=$deposit:expr) => {
+        call_borsh!(
+            $signer,
+            $contract,
+            $method,
+            $arg,
+            near_sdk_sim::DEFAULT_GAS,
+            $deposit
+        )
+    };
+}
+
 #[test]
 fn test_eth_token_transfer() {
     let (user, factory) = setup_token_factory();
     let root = "root".to_string();
```

Comment:
This variable is redundant and is not used.

Reply:
Solved in https://github.com/near/rainbow-token-connector/pull/39/commits/3bd85562f038c258d32019faf7028bee96642790